src/js/component/panel/PanelIssues.js

```diff
@@ -119,25 +119,45 @@ function getOrderedFilteredIssues({
         });
     }
 
-    // Sort by priority, then owner
-    preparedIssues = _.sortBy(preparedIssues, (item) => {
-        const priority = priorities[item.url ?? ''] && (priorities[item.url].priority !== undefined)
+    // Sort by priority, then owner. Sorting has to be chained, since _.sortBy doesn't support multi-criteria sorting.
+    // If an array is returned, it will be sorted lexicographically and cause issues above 10 items.
+    // Iteratee for sorting by owner (secondary sort, for unprioritized items)
+    const ownerSortIteratee = (item) => {
+        const priorityValue = priorities[item.url ?? ''] && (priorities[item.url].priority !== undefined)
```

Comment:
```suggestion
        const priorityValue = priorities[item.url ?? '']?.priority !== undefined;
```
NAB: I think you could use optional chaining here like this

Reply:
We need to update K2 to use a newer eslint config to allow that. Out of scope for this PR I would say.

<img width="586" alt="image" src="https://github.com/user-attachments/assets/767f3b84-d3da-4360-acc6-ff32c68f6809" />